modules/ROOT/pages/questions/question-16.adoc

```diff
@@ -0,0 +1,12 @@
+16. Options in the `link` plugin were re-named. To wit:
```

Comment:
@HAFRMO TINY-4603

Reply:
I can answer this one for Hannes - yes, it was done for clarity and consistency. Options generally follow a naming convention that means they are prefixed with the name of the plugin they belong to. These options did not follow that convention, though it is an odd situation due to them belonging to two plugins. But we wanted to rename them to make it clearer what they were for, anyway.